src/ast.rs

```diff
@@ -1404,6 +1422,62 @@ impl ConcreteNodeMut for BlockStatement {
     }
 }
 
+#[derive(Default, Debug)]
+pub struct BraceHeader {
+    parent: Option<*const dyn Node>,
+    pub kw_lbrace: KeywordLBrace,
+    /// Note that '{' does NOT require a semi or nl afterwards.
```

Comment:
By a historical quirk the same is true for `begin`; it would be good to add a similar comment there.